### PR TITLE
[build] Bump superset-ui packages and update build

### PR DESCRIPTION
**Pull request by etr2460**
### CATEGORY

Choose one

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [x] Build / Development Environment
- [ ] Documentation

### SUMMARY
Recently there's been some build config changes in the superset-ui repos, this pulls the new packages into Superset and updates the build to work appropriately

### TEST PLAN
View a dashboard and see that charts still work as expected
Screen Shot 2020-03-04 at 2 40 38 PM

ensure `npm run prod` and `npm run build` still work

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### REVIEWERS
to: @kristw @graceguo-supercat @rusackas @nytai

### CODE REVIEW

**Comment by kristw** [comment]
Build error related to https://github.com/facebook/jest/issues/8660
Perhaps remove `modules: false` from babel config according to one of the comments.

**Comment by etr2460** [comment]
Yup, just saw that. Hopefully CI passes now

File: superset-frontend/babel.config.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  ],
+  plugins: [
+    'lodash',
+    '@babel/plugin-syntax-dynamic-import',
+    '@babel/plugin-proposal-class-properties',
+    'react-hot-loader/babel',
+    ['@babel/plugin-transform-runtime', { corejs: 3 }],
+  ],
+  env: {
+    test: {
```
Comment (williaster) [inline superset-frontend/babel.config.js]: after all the pain of figuring this out, should we add comments for why the different test config is needed?

File: superset-frontend/spec/javascripts/dashboard/components/FilterTooltipWrapper_spec.jsx [hunk 2]
```diff
@@ -19,6 +19,7 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 import { Overlay, Tooltip } from 'react-bootstrap';
+import { promiseTimeout } from '@superset-ui/core';
```
Comment (kristw) [inline superset-frontend/spec/javascripts/dashboard/components/FilterTooltipWrapper_spec.jsx]: Probably not needed if you are not using it.

**Review by kristw — APPROVED** [review]

File: superset-frontend/jest.config.js [hunk 3]
```diff
@@ -17,6 +17,7 @@
  * under the License.
  */
 module.exports = {
+  timers: 'real',
```
Comment (kristw) [inline superset-frontend/jest.config.js]: not needed. default is `real`. I have tried removing and it works.

File: superset-frontend/spec/javascripts/components/AnchorLink_spec.jsx [hunk 4]
```diff
@@ -18,7 +18,7 @@
  */
 import React from 'react';
 import { shallow } from 'enzyme';
-import sinon from 'sinon';
+import { promiseTimeout } from '@superset-ui/core';
```
Comment (kristw) [inline superset-frontend/spec/javascripts/components/AnchorLink_spec.jsx]: Could use `async` pattern similar to other ones for consistency.

File: superset-frontend/spec/javascripts/dashboard/components/FilterTooltipWrapper_spec.jsx (hunk 2 above)
Comment (etr2460) [reply superset-frontend/spec/javascripts/dashboard/components/FilterTooltipWrapper_spec.jsx]: ugh, yup. idk why a linter didn't catch this :(

File: superset-frontend/spec/javascripts/components/AnchorLink_spec.jsx (hunk 4 above)
Comment (etr2460) [reply superset-frontend/spec/javascripts/components/AnchorLink_spec.jsx]: fair enough

File: superset-frontend/babel.config.js (hunk 1 above)
Comment (etr2460) [reply superset-frontend/babel.config.js]: comment added

**Pull request merged**